UnitTests/UnitTests.UWP/UI/Controls/Test_ImageEx.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Toolkit.Uwp;
+using Microsoft.Toolkit.Uwp.UI;
+using Microsoft.Toolkit.Uwp.UI.Controls;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace UnitTests.UWP.UI.Controls
+{
+    [TestClass]
+    public class Test_ImageEx : VisualUITestBase
+    {
+        [TestMethod]
+        public async Task SetLoadedImage()
+        {
+            await App.DispatcherQueue.EnqueueAsync(async () =>
+            {
+                var bitmapImage = new BitmapImage();
+
+                using var client = new HttpClient();
+                var response = await client.GetAsync(new Uri("https://st.depositphotos.com/1428083/2946/i/600/depositphotos_29460297-stock-photo-bird-cage.jpg"));
```

Comment:
Can we load this from a local resource? As otherwise we depend on this server being up for the CI to run.

Reply:
> 
> 
> Can we load this from a local resource? As otherwise we depend on this server being up for the CI to run.

Maybe we should include an image in the project as a file and use it for both tests.

Reply:
I'd just add one to the Assets folder or just use one of the ones already there:

![image](https://user-images.githubusercontent.com/24302614/117883591-6ff71380-b260-11eb-9999-bee2ed409f20.png)

You should be able to call `Assembly.GetExecutingAssembly().GetManifestResourceStream()` to load it and then put that in the MemoryStream.

We may want a third copy of the test that's like the DataUri one but that just uses ms-appx:// to that resource as well.